### PR TITLE
ac: add Thai translation

### PR DESCRIPTION
**Pull request by nontw**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.th/linux/ac.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`ac`
+
+- แสดงสถิติเวลาที่ผู้ใช้ทุกคนได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง แจกแจงตามผู้ใช้แต่ละคน
+
+`ac --individual-totals`
+
+- แสดงสถิติเวลาของผู้ใช้ที่ระบุไว้ได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง
+
+`ac --individual-totals {{username}}`
```
Comment (navarroaxel) [inline pages.th/linux/ac.md:16]: Please translate the placeholders, like `username`, for get this easier for non-English speakers.

Comment (nontw) [reply pages.th/linux/ac.md]: Yes, I just added the translation as suggested :).

File: pages.th/linux/ac.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# ac
+
+> แสดงสถิติที่ผู้ใช้งานได้เชื่อมต่อเข้ามาในระบบเป็นเวลานานเท่าไหร่
+> ข้อมูลเพิ่มเติม: <https://www.gnu.org/software/acct/manual/accounting.html#ac>.
+
+- แสดงสถิติเวลาที่ผู้ใช้ปัจจุบันได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง
```
Comment (kbdharun) [inline pages.th/linux/ac.md:6]:
```suggestion
- แสดงสถิติเวลาที่ผู้ใช้ปัจจุบันได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง.
```
Or you could use your native equivalent of .

File: pages.th/linux/ac.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`ac`
+
+- แสดงสถิติเวลาที่ผู้ใช้ทุกคนได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง แจกแจงตามผู้ใช้แต่ละคน
```
Comment (kbdharun) [inline pages.th/linux/ac.md:10]:
```suggestion
- แสดงสถิติเวลาที่ผู้ใช้ทุกคนได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง แจกแจงตามผู้ใช้แต่ละคน:
```

File: pages.th/linux/ac.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`ac --individual-totals`
+
+- แสดงสถิติเวลาของผู้ใช้ที่ระบุไว้ได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง
```
Comment (kbdharun) [inline pages.th/linux/ac.md:14]:
```suggestion
- แสดงสถิติเวลาของผู้ใช้ที่ระบุไว้ได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง:
```

File: pages.th/linux/ac.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`ac --individual-totals {{ชื่อผู้เข้าใช้}}`
+
+- แสดงสถิติว่าผู้ใช้ที่ระบุไว้ ได้เชื่อมต่อเข้ามาในรบบเป็นเวลากี่ชั่วโมงต่อวัน (พร้อมทั้งแสดงเวลาทั้งหมด)
```
Comment (kbdharun) [inline pages.th/linux/ac.md:18]:
```suggestion
- แสดงสถิติว่าผู้ใช้ที่ระบุไว้ ได้เชื่อมต่อเข้ามาในรบบเป็นเวลากี่ชั่วโมงต่อวัน (พร้อมทั้งแสดงเวลาทั้งหมด):
```

File: pages.th/linux/ac.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`ac --daily-totals --individual-totals {{ชื่อผู้เข้าใช้}}`
+
+- แสดงข้อมูลเพิ่มเติมต่างๆ
```
Comment (kbdharun) [inline pages.th/linux/ac.md:22]:
```suggestion
- แสดงข้อมูลเพิ่มเติมต่างๆ:
```

File: pages.th/linux/ac.md (hunk 2 above)
Comment (kbdharun) [inline pages.th/linux/ac.md:6]:
```suggestion
- แสดงสถิติเวลาที่ผู้ใช้ปัจจุบันได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง:
```

**Comment by nontw** [comment]
Incorporated all the  suggestions.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**